episodes/anonymization.md

```diff
@@ -32,19 +33,352 @@ Metadata elements in imaging, such as patient names and addresses, are often cle
 
 DICOM files contain metadata, which includes various types of identifying information that should remain confidential. The easiest way to mitigate issues with DICOM metadata is to avoid having it in the first place. If possible, opt to receive just the images and select metadata rather than the entire DICOM file. When sharing data with collaborators, there is often no need to share the full DICOM files.
 
+
+### Text on Images
+
+Occasionally, technicians will burn information directly onto images as part of a burned-in annotation. This may include details such as diagnoses, demographics, or the patient's name. Fortunately, this text is usually typed rather than handwritten, making it recognizable by optical character recognition (OCR) functions. Often, this text is placed away from the center of the image, allowing for clever cropping to eliminate it entirely in some datasets.
+
+
+
+::::::::::::::::::::::::::::::::::::::: challenge
+
+## Challenge: Getting rid of identifying burned in data
+
+An ultrasound (scraped from the public internet on a creative commons license lisence [here](https://www.flickr.com/photos/jcarter/2461223727).) must be kept at it's existing height and width dimensions. You should change the image from RGB to grayscale. It can be opened as follows:
```

Comment:
Oh, and "its existing"

Reply:
Looking at this again, I think it needs more text. As stated, the assignment is to not change the dimensions of the image (easy, do nothing) and convert it to grayscale.

I've seen the title, but it would read more easily if this were along the lines of "Download [this ultrasound] and have a look at it. Remove any burned-in identifying characteristics without changing the height or width of the image."